docs/DeveloperGuide.md

```diff
@@ -6,33 +6,183 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
-
+The sequence diagrams for our main class, CantVasMain and its interactions with
+other classes.
+![img.png](img.png)
+![img_1.png](img_1.png)
```

Comment:
Would look more professional if the diagrams were drawn with PlantUML instead of hand drawn